isample/.htaccess

```
@@ -0,0 +1,4 @@
+Options +FollowSymLinks
+RewriteEngine on 
+
+RewriteRule       ^$ | ^/$            https://isamplesorg.github.io/home/ [R=303,L]
```

Is the `/` part even needed for root things like this?  In any case, if you need to handle `/` specifically here it may be easier just to make it optional:
```suggestion
RewriteRule       ^/?$            https://isamplesorg.github.io/home/ [R=303,L]
```